app/javascript/components/ProductEdit/Layout.tsx

```diff
@@ -31,6 +34,82 @@ export const useProductUrl = (params = {}) => {
       });
 };
 
+const NotifyAboutProductUpdatesAlert = () => {
+  const { uniquePermalink, contentUpdates, setContentUpdates } = useProductEditContext();
+  const timerRef = React.useRef<number | null>(null);
+  const isVisible = !!contentUpdates;
+
+  const clearTimer = () => {
+    if (timerRef.current !== null) {
+      clearTimeout(timerRef.current);
+      timerRef.current = null;
+    }
+  };
+
+  const startTimer = () => {
+    clearTimer();
+    timerRef.current = window.setTimeout(() => {
+      close();
+    }, 10_000);
+  };
+
+  const close = () => {
+    clearTimer();
+    setContentUpdates(null);
+  };
+
+  React.useEffect(() => {
+    if (isVisible) {
+      startTimer();
+    }
+
+    return clearTimer;
+  }, [isVisible]);
+
+  const handleMouseEnter = () => {
+    clearTimer();
+  };
+
+  const handleMouseLeave = () => {
+    startTimer();
+  };
+
+  return (
+    <div
+      role="alert"
+      className={cx("fixed right-1/2 top-4", "info", isVisible ? "visible" : "invisible")}
+      style={{
+        transform: `translateX(50%) translateY(${isVisible ? 0 : "calc(-100% - var(--spacer-4))"})`,
+        transition: "all 0.3s ease-out 0.5s",
+        zIndex: "var(--z-index-tooltip)",
+        backgroundColor: "var(--body-bg)",
```

Comment:
Does this override the background color completely?

We should keep the original look and feel while preventing other elements in the background from leaking through.

I was suggesting having a parent container with the solid color `var(--body-bg)` background, so that the nothing else could leak through the actual alert element itself.

Reply:
Thanks for making the change!

FYI a screenshot like below could really be helpful and speed up the reviews!

<img width="688" alt="image" src="https://github.com/user-attachments/assets/a9574f21-2d22-4066-a3cf-cdc53263d686" />